src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import App from './App';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './states';
+import './styles/style.css';
+const root = createRoot(document.getElementById('root'));
+root.render(
+<Provider store={store}>
+    <BrowserRouter>
+            <App/>
+    </BrowserRouter>
```

Comment:
```suggestion
    <BrowserRouter>
                 <React.StrictMode>
                          <App/>
                 </React.StrictMode>
    </BrowserRouter>
```
Kamu belum menerapkan React Strict Mode yang merupakan salah satu point dari kriteria utama no 2 bugs highlighting. Dengan menerapkan React Strict Mode potensi bugs akan lebih mudah terdeteksi.